src/ucs/sys/topo/base/topo.c

```diff
@@ -462,28 +587,32 @@ void ucs_topo_print_info(FILE *stream)
     }
 }
 
-static ucs_sys_topo_method_t ucs_sys_topo_sysfs_method = {
-    .name         = "sysfs",
-    .get_distance = ucs_topo_get_distance_sysfs,
+static ucs_sys_topo_provider_t ucs_sys_topo_provider_sysfs = {
+    .name = "sysfs",
+    .ops =
+            {
```

Comment:
weird alignment

Reply:
AS far as I remember all alignment in topo.c are what clang suggested...

Reply:
@ofirfarjun7  let's stick with the way it is done in similar places and looks more natural, even if contradicts `clang-format`.